spec/classes/github_actions_runner_spec.rb

```diff
@@ -49,6 +49,34 @@
         end
       end
 
+      context 'is expected to contain archive' do
+        it do
+          is_expected.to contain_archive("first_runner-test_package-1.0.1.tar.gz").with({
+            'ensure' => 'present',
+            'user'   => 'test_user',
+            'group'  => 'test_group',
+          })
+        end
+      end
+
```

Comment:
Can it be added a new file (i.e. `spec/classes/github_actions_runner_default_values_spec.rb`) for testing the default paremeters ? 
Here it is being tested with a user and group `test_user` and `test_group` among other class parameters. Default parameters should be also checked

Reply:
ok - ive reverted all test to be with default params 
and added 2 new tests for user and group